app/views/UkRevenueNotEligibleView.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        layout: templates.Layout,
+        govukButton: GovukButton
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("globalRevenuesNotEligible.title"))) {
+
+    <h1 class="govuk-heading-l">@messages("globalRevenuesNotEligible.heading")</h1>
+    <p class="govuk-body">@messages("ukRevenuesNotEligible.p1")</p>
+    <p class="govuk-body">@HtmlContent(messages("globalRevenuesNotEligible.change-answers", "/digital-services-tax-registration")).asHtml</p>
```

Comment:
"go back and change your answers" link needs to go to the uk revenues page {/uk-revenues}